index.html

```diff
@@ -21,6 +22,14 @@
     <!-- Main page content -->
     <main>
       <h1>Latest topics</h1>
+      <table>
```

Comment:
- I'd add `<thead>` grouping element for the header row. Not only it describes the part of the table, semantic value, but it can be used for styling without creating new class names.

- I also noticed that when is the `<thead>` element omitted, the browser will add the row automatically inside `<tbody`> even though it's not been defined.

- I'd also give our table an `id` attribute for styling or further work. Moreover, we're going to implement `<tbody>` in the next issue.

```
      <table id="forum-table">
        <thead>
          <tr>
            <th class="topics">Topics</th>
            <th class="replies">Replies</th>
            <th class="views">Views</th>
            <th class="activity">Activity</th>
          </tr>
        </thead>
      </table>
```